scrapers/NaughtyNatural.yml

```diff
@@ -0,0 +1,54 @@
+name: Naughty Natural
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - naughtynatural.com/videos/
+    scraper: sceneScraper
+galleryByURL:
+  - action: scrapeXPath
+    url:
+      - naughtynatural.com/photos/
+    scraper: galleryScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $url: //link[@rel="canonical"]/@href
+    scene:
+      Title: &title //h1[@class="entry-title"]/text()
+      Studio: &studio
+        Name:
+          fixed: Naughty Natural
+      Date: &date
+        selector: //meta[@property="article:published_time"]/@content
+        postProcess:
+          - replace:
+              - regex: (^\d{4}-\d{2}-\d{2}).*
+                with: $1
+          - parseDate: 2006-01-02
+      Details:
+        selector: //p[@class="description"]/text()
+      Tags: &tags
+        Name: //span[@class="categories"]/a
+      Performers: &performers
+        Name: //span[@class="post_author"]/a
+      # Unable to scrape image! Xpath correctly isolates jpg file, however AWS server blocks scraping of image
+      #Image:
+      #  selector: //script[@type="text/javascript"][contains(text(),"jwplayer")]
+      #  postProcess:
+      #    - replace:
+      #        - regex: .*(https.+jpg).*
+      #          with: $1
```

Comment:
I noticed that while they are using AWS to store these images, they do also store it right on their CMS as well... we can do heinous regex stuff to grab it from there instead! If we grab the path from the first URL (the image) and the domain from the second URL (the video) and Frankenstein them together then it resolves and we can scrape the image

```
Image:
  selector: //script[@type="text/javascript"][contains(text(),"jwplayer") and contains(text(),"jpg")]
  postProcess:
    - replace:
        - regex: .*(wp-content.+jpg).*(https:\/\/[^\/]*).*
          with: $2/$1
```

Reply:
Perfect. Done.